pages/deployment/[uuid].tsx

```diff
@@ -96,37 +97,28 @@ export default function Deployment() {
 
   const [group, setGroup] = useState<TabOptions>('overview')
   const [complianceFlow, setComplianceFlow] = useState<Elements>([])
-  const [open, setOpen] = useState<boolean>(false)
-  const [tag, setTag] = useState<string>('')
+  const [open, setOpen] = useState(false)
   const [anchorEl, setAnchorEl] = useState<HTMLDivElement | null>(null)
+  const [imageName, setImageName] = useState('')
+  const [selectedImageTag, setSelectedImageTag] = useState('')
+
   const actionOpen = anchorEl !== null
 
   const { currentUser, isCurrentUserLoading, isCurrentUserError } = useGetCurrentUser()
   const { deployment, isDeploymentLoading, isDeploymentError, mutateDeployment } = useGetDeployment(uuid, true)
   const { uiConfig, isUiConfigLoading, isUiConfigError } = useGetUiConfig()
+  const model: ModelDoc = deployment?.model as ModelDoc
+  const { versions } = useGetModelVersions(model?.uuid)
```

Comment:
If we can guarantee this is a ModelDoc in the UI and this is just a type issue then we can simplify it slightly `const { versions } = useGetModelVersions((deployment?.model as ModelDoc).uuid)`

Reply:
Unfortunately returns undefined!